site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/rsf/GroupImportStep2Producer.java

```diff
@@ -91,7 +91,7 @@ public void fillComponents(UIContainer tofill, ViewParameters viewParams, Compon
             Group existingGroup = null;
             
             //add title
-            UIOutput.make(branch,"title",importedGroup.getGroupTitle());
+            UIOutput.make(branch,"title","Group: " + importedGroup.getGroupTitle());
```

Comment:
Shouldn't this be pulled from the `messageLocator`?

Reply:
Code changed to use messageLocator.